client/app/bundles/shared/components/Footer/Footer.scss

```diff
@@ -0,0 +1,133 @@
+@import '../base';
+
+.footer {
+  width: 100%;
+  height: 450px;
```

Comment:
Let's actually take out the fixed height so the container can resize based on its children. This causes the "We <3 FOSS" copy to overlap a bit with the bottom part of the container as well, so if you could fix that up, that would be great!